docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-cloudformation-template-with-secured-sumoendpoint.md

```diff
@@ -0,0 +1,123 @@
+---
+id: collect-with-cloudformation-template-with-secured-sumoendpoint
+title: Collect Amazon CloudWatch Logs using a CloudFormation Template with Secured Endpoint
+sidebar_label: Collect Logs using CloudFormation with Secured Endpoint
+description: Learn how to collect Amazon CloudWatch Logs using a CloudFormation template with secured Sumo EndPoint.
+---
+:::note
+We strongly recommend the alternative collection process described on [AWS Kinesis Firehose for Logs Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/), which is more robust and reliable, where you do not need to manage resources. Lambda based collection methods are limited by time out, concurrency, and memory limits.
+:::
+
+This method is similar to [Collect CloudWatch Logs using a CloudFormation Template](/docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-cloudformation-template) the only difference is that this method stores the Sumo Logic Endpoint securely in system manager.
```

Comment:
I don't think **Endpoint** needs to be capitalized unless it's like that in the UI

Reply:
Yes @kimsauce, it should not be capitalised. I will update it.